Update edge-runtime.md (#38271)
vercel/next.js: docs/api-reference/edge-runtime.md

You can use `process.env` to access [Environment Variables](/docs/basic-features/environment-variables.md) for both `next dev` and `next build`.

> Running `console.log` on `process.env` will not show all your environment variables, you have to access the variables directly before they are shown in `process.env`

Reviewer: ```suggestion
Running `console.log` on `process.env` **will not** show all your Environment Variables. You have to access the variables directly as shown below:
```
